README.md

```diff
@@ -171,6 +171,111 @@ See the [detailed query manager docs][query-manager-api] for more details on
 usage, or the [Apollo service API][apollo-service-api] if you need to use
 the service directly.
 
+### GraphQL Subscriptions
+
+GQL Subscriptions allow a client to subscribe to specific queries they are interested in tracking. The syntax for doing this is similar to `query` / `watchQuery`, but there are a few main differences:
+
+- ) you must define a `subscription` (versus a `query` or `mutation`)
+- ) because subscriptions are async by nature, you have to listen for these events and act accordingly.
+- ) subscriptions require websockets, so must configure your `link` accordingly
+
+**Creating your subscription:**
+
+`my-app/gql/subscription/new-human`
+
+```
+subscription {
+  newHuman() {
+    name
+  }
+}
+```
+
+**Subscribing from inside a route:**
+
+`app/routes/some-route.js`
+
+```js
+import Route from "@ember/routing/route";
+import { RouteQueryManager } from "ember-apollo-client";
+import query from "my-app/gql/subscription/new-human";
+
+export default Route.extend(RouteQueryManager, {
+  model() {
+    return this.get("apollo")
+               .subscribe({ query }, "human");
+  },
+
+  setupController(controller, model) {
+    model.on("event", event => alert(`${event.name} was just born!`));
+  },
+});
+```
+
+The big advantage of using the RouteQueryManager is that when you navigate away from this route, all subscriptions created will be terminated. That said, if you want to manually unsubscribe (or are not using the RouteQueryManager) `subscription.unsubscribe()` will do the trick.
```

Comment:
can you put each use of `RouteQueryManager` in backticks since it's a class name?